src/json-ld.net/Core/Context.cs

```diff
@@ -14,15 +14,12 @@ namespace JsonLD.Core
 	/// <author>tristan</author>
 	//[System.Serializable]
 	public class Context : JObject
-#if !PORTABLE && !IS_CORECLR
-        , ICloneable
-#endif
     {
         private JsonLdOptions options;
 
 		JObject termDefinitions;
 
-		public JObject inverse = null;
+		internal JObject inverse = null;
```

Comment:
This is the sort of changes from #77 we don't want, no?

Reply:
thanks for catching. I guess I screwed up the cherry-pick. Will fix this Monday.

Reply:
....or tuesday....